test_suite/derive_tests_no_std/Cargo.toml

```diff
@@ -11,8 +11,13 @@ license = "Apache-2.0"
 
 [dependencies]
 scale-info = { path = "../..", default-features = false, features = ["derive"] }
-scale = { package = "parity-scale-codec", version = "2", features = ["full"] }
```

Comment:
does it work with ` scale = { package = "parity-scale-codec", version = "2", default-features = false }`?

Or why was it removed?

Reply:
I just removed it from the dependencies because it was not used before and is not needed.